data/kafka-connector-published-branches.yaml

```diff
@@ -0,0 +1,16 @@
+version:
+  published:
+    - '0.1'
```

Comment:
This list has to be the same length as `git.branches.published`, doesn't it?

Reply:
seemingly not, but it would make sense if it were…

Reply:
Updating to just use master for now, since apparently it's unclear what the first release edition will be anyway.